core/test/gather/driver/wait-for-condition-test.js

```diff
@@ -63,7 +63,7 @@ describe('waitForFullyLoaded()', () => {
 
   beforeEach(() => {
     session = {sendCommand: fnAny().mockResolvedValue(), setNextProtocolTimeout: fnAny()};
-    networkMonitor = {};
+    networkMonitor = {getInflightRequests: fnAny().mockReturnValue([])};
 
```

Comment:
What's the recommended approach to mocking `log`  to test if it has been called correctly? I found `importMock`, but it doesn't seem to support mocking external modules.

Reply:
`log.events` will emit events when something is logged, so you can collect those events and assert those. One example:

https://github.com/GoogleChrome/lighthouse/blob/2b8caf21a3e9d03633b52e43e35e97291ae6553d/core/test/config/filters-test.js#L471-L482

Reply:
Let me know if the test is fine or you would like something more and I'll add it :D  
`codecov` seems to be complaining for some reason though as if it didn't catch the test :/